spec/features/idv/doc_auth/choose_id_type_spec.rb

```diff
@@ -5,30 +5,35 @@
   include AbTestsHelper
   include IdvStepHelper
 
+  let(:ipp_service_provider) do
```

Comment:
I did some clean up moving as much of the setup to the top so there are small, clear context setup below.